library/middleware/koa.ts

```diff
@@ -0,0 +1,37 @@
+import { shouldBlockRequest } from "./shouldBlockRequest";
+import { escapeHTML } from "../helpers/escapeHTML";
+import type Application = require("koa");
```

Comment:
not possible with

`import type * as Application from "koa"`? :D

Reply:
Ahh, yes 😅. Typescript always recommended this when trying to import with `import type { Application } from "koa"`.